raiden/ui/cli.py

```
@@ -392,55 +394,62 @@ def run(ctx, **kwargs):
                     )
-                )
+                    (api_host, api_port) = split_endpoint(kwargs["api_address"])
+                    api_server.start(api_host, api_port)
 
-            if ctx.params['console']:
-                console = Console(app_)
-                console.start()
-
-            # wait for interrupt
-            event = gevent.event.Event()
-            gevent.signal(signal.SIGQUIT, event.set)
-            gevent.signal(signal.SIGTERM, event.set)
-            gevent.signal(signal.SIGINT, event.set)
-            event.wait()
+                    print(
+                        "The Raiden API RPC server is now running at http://{}:{}/.\n\n"
+                        "See the Raiden documentation for all available endpoints at\n"
+                        "https://github.com/raiden-network/raiden/blob/master"
+                        "/docs/Rest-Api.rst".format(
+                            api_host,
+                            api_port,
+                        )
+                    )
 
-            try:
-                api_server.stop()
-            except NameError:
-                pass
+                if ctx.params['console']:
+                    console = Console(app_)
+                    console.start()
+
+                # wait for interrupt
+                event = gevent.event.Event()
+                gevent.signal(signal.SIGQUIT, event.set)
+                gevent.signal(signal.SIGTERM, event.set)
+                gevent.signal(signal.SIGINT, event.set)
+                event.wait()
+
+                try:
+                    api_server.stop()
+                except NameError:
+                    pass
+        except socket.error as v:
+            if v.args[0] == errno.EADDRINUSE:
+                print("ERROR: %s:%s is in use. "
+                      "Use --listen-address <host:port> to specify port to listen on." %
+                      (listen_host, listen_port))
```

Ah sorry another nitpick: In case there is a socket error that isn't "address already in use" this currently swallows the exception and exits silently. I'd say just re-raise in the `else` case since those should be pretty unlikely edge cases.